docs/about/releases.mdx

```diff
@@ -15,6 +15,8 @@ description: "Latest new features and changelog."
 
 [All releases](https://github.com/mage-ai/mage-ai/releases)
 
+
```

Comment:
@thomaschung408 You can remove this extra newline here. May make the docs page have a bit top much space with it.